tutorials/physics/using_jolt_physics.rst

```diff
@@ -0,0 +1,473 @@
+- ``collisions/soft_body_point_margin`` is now ``simulation/soft_body_point_radius.``
+- ``collisions/body_pair_cache_enabled is now simulation/body_pair_contact_cache_enabled.``
+- ``collisions/body_pair_cache_distance_threshold`` is ``now simulation/body_pair_contact_cache_distance_threshold.``
+- ``collisions/body_pair_cache_angle_threshold is now simulation/body_pair_contact_cache_angle_threshold.``
+- ``continuous_cd/movement_threshold`` is now ``simulation/continuous_cd_movement_threshold``,
+  but expressed as a fraction instead of a percentage.
+- ``continuous_cd/max_penetration`` is now ``simulation/continuous_cd_max_penetration``,
+  but expressed as a fraction instead of a percentage.
+- ``kinematics/use_enhanced_internal_edge_removal`` is now ``motion_queries/use_enhanced_internal_edge_removal.``
+- ``kinematics/recovery_iterations`` is now ``motion_queries/recovery_iterations``,
+  but expressed as a fraction instead of a percentage.
+- ``kinematics/recovery_amount`` is now ``motion_queries/recovery_amount.``
+- ``queries/use_legacy_ray_casting`` has been removed.
+- ``solver/position_iterations`` is now ``simulation/position_steps.``
+- ``solver/velocity_iterations`` is now ``simulation/velocity_steps.``
+- ``solver/position_correction`` is now ``simulation/baumgarte_stabilization_factor``,
+  but expressed as a fraction instead of a percentage.
+- ``solver/active_edge_threshold`` is now ``collisions/active_edge_threshold.``
+- ``solver/bounce_velocity_threshold`` is now ``simulation/bounce_velocity_threshold.``
+- ``solver/contact_speculative_distance`` is now ``simulation/speculative_contact_distance.``
+- ``solver/contact_allowed_penetration`` is now ``simulation/penetration_slop.``
+- ``limits/max_angular_velocity`` is now stored as radians instead.
+- ``limits/max_temporary_memory`` is now ``limits/temporary_memory_buffer_size.``
+
+There might be some discussion to be had with regards to migrating the settings
+values for projects who have previously been relying on the extension.
+
+Joint Nodes
+~~~~~~~~~~~
+
+The joint nodes that are exposed in the Godot Jolt extension (JoltPinJoint3D,
+JoltHingeJoint3D, JoltSliderJoint3D, JoltConeTwistJoint3D and JoltGeneric6DOFJoint)
+have not been included with this module.
+
+Thread Safety
+~~~~~~~~~~~~~
+
+Unlike the Godot Jolt extension, this module does have experimental thread-safety,
+including support for the :ref:`physics/3d/run_on_separate_thread<class_ProjectSettings_Property_physics/3d/run_on_separate_thread>`
+project setting. This is achieved by utilizing the same wrapper server that's used
+by Godot Physics, called ``PhysicsServer3DWrapMT``, as well as introducing a mutex
+around the rebuilding of shapes, since concurrent shape-casts could otherwise
+trigger a race condition.
+
+This has however not been tested very thoroughly, so should be considered
+experimental.
+
+Query Performance
```

Comment:
I think this has been fixed by https://github.com/godotengine/godot/pull/101071 so this section can probably be removed.